src/components/ChatEntry.js

```diff
@@ -1,22 +1,35 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = ({ id, sender, body, timeStamp, liked, onToggleHeart }) => {
+  const displaySide = sender === 'Vladimir' ? 'local' : 'remote';
```

Comment:
How could we dynamically get the names of the chat participants? What if the chat were just starting and there weren't any messages yet? And what if our chat data were to change and the messages were between Mario and Luigi (or literally any other name lol) ?